programs/zstd.1

```diff
@@ -1,455 +0,0 @@
-.
```

Comment:
It seems `zstd.1` has been emptied in the process.
Maybe a local issue with the man page builder ?

Not a huge issue, I can rebuild it on my local dev station if necessary.